SU2_CFD/src/solvers/CEulerSolver.cpp

```diff
@@ -13408,3 +13897,246 @@ void CEulerSolver::GatherInOutAverageValues(CConfig *config, CGeometry *geometry
     }
   }
 }
+
+void CEulerSolver::ComputeTurboPerformance(CConfig *config, CGeometry *geometry) {
+
+  unsigned short nDim = geometry->GetnDim();
+  unsigned short iMarkerTP, iSpan, iDim, iStage, iBlade;
+  unsigned short nMarkerTP = config->GetnMarker_Turbomachinery();
+  su2double area, absVel2, soundSpeed, mach, tangVel, tangVel2, *relVel, relVel2;
+  su2double relPressureIn, relPressureOut, enthalpyOutIs, relVelOutIs2;
+  relVel = new su2double[nDim];
+  su2double muLam, kine, omega, nu;
+  bool turbulent = ((config->GetKind_Solver() == RANS) || (config->GetKind_Solver() == DISC_ADJ_RANS));
+  bool menter_sst       = (config->GetKind_Turb_Model() == SST);
```

Comment:
Beware that there is also SST_SUST now, might not apply to turbomachinery but... it exists.

Reply:
Honestly I did not know, thanks for pointing it out.